doc/theoretical_description_binary_classification.rst

```diff
@@ -6,41 +6,21 @@
 Theoretical Description
 =======================
 
```

Comment:
Global remark : in this part, to talk about the model we use $f$ while everywhere else in the documentation we talk about $\hat{\mu}$. We should uniformize

Reply:
Done.